packages/ratatouille/PKGBUILD

```diff
@@ -0,0 +1,104 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=Ratatouille.lv2
+pkgbase=ratatouille
+pkgname=(ratatouille ratatouille-{clap,lv2,standalone,vst})
+pkgver=0.9.11
+pkgrel=2
+pkgdesc='Guitar amp neural model and impulse response loader and mixer'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name"
+license=(BSD-3-Clause)
+groups=(pro-audio)
+depends=(glibc gcc-libs libx11)
+makedepends=(cairo jack libsndfile lv2 xxd)
+checkdepends=(lv2lint)
+source=("https://github.com/brummer10/$_name/releases/download/v$pkgver/$_name-v$pkgver-src.tar.xz")
+sha256sums=('9a8134a056b6ba62144dd49ba73b1acc9ae1015868d735ba2da4b94feeb707a1')
+_plugin_uri="urn:brummer:ratatouille"
+
+_pick() {
+  local p="$1" f d; shift
+  for f; do
+    d="$srcdir/$p/${f#$pkgdir/}"
+    mkdir -p "$(dirname "$d")"
+    mv "$f" "$d"
+    rmdir -p --ignore-fail-on-non-empty "$(dirname "$f")"
+  done
+}
+
+build() {
+  cd $_name-v$pkgver
+  # https://github.com/brummer10/Ratatouille.lv2/issues/42
+  export CXXFLAGS="${CXXFLAGS/-Wp,-D_FORTIFY_SOURCE=3/}"
+  make CXX=g++ STRIP=:
+}
+
+check() {
+  cd $_name-v$pkgver
+  lv2lint -M pack -s '_Z*' -I bin/$_name "$_plugin_uri"
+}
+
+package_ratatouille() {
```

Comment:
Not sure but does it make sense to set `arch=(any)` here since it's just a meta package?